WordPressComKit/MeService.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import Alamofire
+
+public class MeService {
+    public init() {}
+    
+    public func fetchMe(completion: (Me?, NSError?) -> Void) {
+        Alamofire
+            .request(RequestRouter.Me())
+            .validate()
+            .responseJSON { response in
+                guard response.result.isSuccess else {
+                    completion(nil, response.result.error)
+                    return
+                }
+                
+                let json = response.result.value as? [String: AnyObject]
+                let me = Me()
+                me.ID = json!["ID"] as! Int
+                me.displayName = json!["display_name"] as? String
+                me.username = json!["username"] as! String
+                me.email = json!["email"] as? String
+                me.primaryBlogID = json!["primary_blog"] as? Int
+                if let primaryBlogURLPath = json!["primary_blog_url"] as? String {
+                    me.primaryBlogURL = NSURL(string: primaryBlogURLPath)
+                }
```

Comment:
We'd need a common way of parsing JSON, so we don't have to duplicate stuff like this all over the place.

In this case there's a subtle bug: `primaryBlogURL == nil` means that either
- :white_check_mark:  `primary_blog_url` was `nil`.
- :x: `primary_blog_url` was an invalid URL. This should raise an error